src/fides/api/ops/service/connectors/email_connector.py

```diff
@@ -56,11 +60,12 @@ def test_connection(self) -> Optional[ConnectionTestStatus]:
 
         try:
             # synchronous for now since failure to send is considered a connection test failure
-            dispatch_email(
+            dispatch_message(
                 db=db,
-                action_type=EmailActionType.EMAIL_ERASURE_REQUEST_FULFILLMENT,
-                to_email=config.test_email,
-                email_body_params=[
+                action_type=MessagingActionType.MESSAGE_ERASURE_REQUEST_FULFILLMENT,
+                to_identity=Identity(**{"email": config.test_email}),
```

Comment:
Do you need the dictionary unpacking here?

```suggestion
                to_identity=Identity(email=config.test_email),
```